contentScripts/googleVoiceManager.js

```diff
@@ -139,6 +139,10 @@ class GoogleVoiceSiteManager {
     }
 
     fillNumberInput() {
+        // Confirm that phone number is not already populated in case this is a retry attempt
+        if (this.confirmChatSwitched()) {
```

Comment:
currently, if this goes into a retry, it will fail at this step because the number is probably already populated.  we want to just skip this step if that's the case so we can proceed on to the confirmSent method, which is a more reliable indicator of whether they're being throttled.